enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,983 @@
+```yaml
+spec:
+  endpointPublishingStrategy:
+    loadBalancer:
+      dnsManagementPolicy: Managed
+      providerParameters:
+        aws:
+          networkLoadBalancer:
+            eipAllocations:
+            - eipalloc-0956fea34de4cb7ab
+            - eipalloc-0e9a3077a70de050a
+            - eipalloc-0b69fc4691f54cdd0
+            - eipalloc-01e6ba6cbba1a391b
+            - eipalloc-0e242df173f906112
+          type: NLB
+        type: AWS
+      scope: External
+    type: LoadBalancerService
+```
+
+Check if the router-default service was created with proper annotations.
+
+```yaml
+% oc get svc router-default -n openshift-ingress -o yaml         
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-eip-allocations: eipalloc-0956fea34de4cb7ab,eipalloc-0e9a3077a70de050a,eipalloc-0b69fc4691f54cdd0,eipalloc-01e6ba6cbba1a391b,eipalloc-0e242df173f906112
+...
+  type: LoadBalancer
+status:
+  loadBalancer:
+    ingress:
+    - hostname: a8491585620154d5992dcdecc06c5779-7938285b62f7bebe.elb.us-east-1.amazonaws.com
+```
+
+2. What happens when someone updates the `eipAllocations` directly in the load balancer service of an Ingress Controller ?
+  It is not supported to directly edit the `eipAllocations` for the load balancer service of an IngressController, and this will put the IngressController into an error state with the following status:
+```yaml
+  - lastTransitionTime: "2024-04-17T03:09:32Z"
+    message: |-
+      The service-controller component is reporting SyncLoadBalancerFailed events like: 
+      Error syncing load balancer: failed to ensure load balancer: error creating load balancer: 
+      "ResourceInUse: The allocation IDs are not available for use\n\tstatus code: 400, 
+      request id: 58136c42-e1bd-4c1a-ac38-7078da2b1d95"
+      The kube-controller-manager logs may contain more details.
+    reason: SyncLoadBalancerFailed
+    status: "False"
+    type: LoadBalancerReady
+  - lastTransitionTime: "2024-04-17T02:54:25Z"
+    message: LoadBalancer is not progressing
+    reason: LoadBalancerNotProgressing
+    status: "False"
+    type: LoadBalancerProgressing
+...
+...
+  - lastTransitionTime: "2024-04-17T03:09:32Z"
+    message: |-
+      One or more status conditions indicate unavailable: LoadBalancerReady=False 
+      (SyncLoadBalancerFailed: The service-controller component is reporting 
+      SyncLoadBalancerFailed events like: Error syncing load balancer: failed to ensure
+      load balancer: error creating load balancer: "ResourceInUse: The allocation IDs 
+      are not available for use\n\tstatus code: 400, request id: 58136c42-e1bd-4c1a-ac38-7078da2b1d95"
+      The kube-controller-manager logs may contain more details.)
+    reason: IngressControllerUnavailable
+    status: "False"
+    type: Available
+
+```
+
+3. Are there any pre-requisites for this EP ?
+  Yes, please refer section `Pre-requisites` under [Workflow Description](#Workflow Description)
```

Comment:
`[Workflow Description]` is an invalid link. I think you need to replace ` ` with `-`.

```suggestion
  Yes, please refer section `Pre-requisites` under [Workflow Description](#workflow-description)
```

Reply:
done